Task6/IRepository.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Task6
+{
+    public interface IRepository
+    {
```

Comment:
folderPath has to be a part of  a repository, not send to Save or Load methods (for consistency). 
Change to void Save(Catalog catalog);